crates/nu-command/src/formats/from/toml.rs

```diff
@@ -80,7 +81,10 @@ fn convert_toml_to_value(value: &toml::Value, span: Span) -> Value {
             span,
         ),
         toml::Value::String(s) => Value::string(s.clone(), span),
-        toml::Value::Datetime(d) => Value::string(d.to_string(), span),
+        toml::Value::Datetime(d) => Value::date(
+            chrono::DateTime::from_str(&d.to_string()).expect("from toml date conversion"),
```

Comment:
Is it possible to have neither unwrap() nor expect() here to keep from panicking? Chrono is super susceptible to panics and we try to avoid them at all costs. Sorry I didn't say that earlier.

Reply:
What would be the optimal way if the result is not available? Because I don't really see what could be done if the to_string() goes wrong.

Reply:
Since it needs to be a date, I think we have to supply a sentinel value.

Reply:
Feel free to think about this a little more. If we can have a date based on what is provided, then that's fine too. So, the question is, in the small case where to_string fails, would it be better to have the date represented as a string in the toml, or would it be better to have a sentinel value? I'm good with whatever direction you want to go.

Reply:
I just added the fix for error case, I decided to return the default date instead of a valid string of some kind since this will make piping this command less of an inconvenience in my opinion.